modules/addons/openprovider/Controllers/Hooks/RenewHookController.php

```diff
@@ -95,21 +95,15 @@ public function processRenewal($params)
             try {
                 $openprovider_domain = $this->openProvider->api->modifyScheduledTransferDate($openprovider_api_domain, date("Y-m-d H:i:s"));
             } catch (\Exception $ex) {
-                // return false;
-                return array(
-                    'abortWithError' => $ex->getMessage(),
-                );
+                return false;
             }
 
             return array (
                 'abortWithSuccess' => true,
             );
         }
 
-        // return false;
-        return array(
-            'success' => true,
-        );
+        return false;
```

Comment:
bit wondering why returning `success` earlier and now it's false, should be `true` right? check it out.
but anyway I'm approving as per code level

Reply:
It was generating a conflict when there was a renewal process from another reseller. @avishka-ra and I tested it and it is now working properly.